rmc/utils/mpc.py

```diff
@@ -405,16 +378,12 @@ def prepare_pop_path_ht(
     # If OE-annotated VEP context HT filtered to missense variants
     # in canonical transcripts does not exist, create it
     # If HT exists but without specified version of OE annotation, add that OE annotation
-    rmc_cols = {
-        "transcript": "transcript_rescue" if include_rescue else "transcript",
-        "oe": "oe_rescue" if include_rescue else "oe",
-    }
+    rmc_cols = {"transcript", "oe"}
     if not file_exists(context_with_oe_dedup.path) or (
         len(set(rmc_cols.values()).intersection(context_with_oe_dedup.ht().row))
         < len(set(rmc_cols.values()))
     ):
         create_context_with_oe(
-            include_rescue=include_rescue,
             overwrite_temp=overwrite_temp,
             overwrite_output=overwrite_output,
         )
```

Comment:
Can't comment but lines 393-397 should be deleted

Reply:
<img width="989" alt="image" src="https://user-images.githubusercontent.com/19142091/216076216-5f21fc6d-4137-426f-8782-61befe7b9ce7.png">
this part? that's just a checkpoint though, right?